doc/usage/configuration.rst

```diff
@@ -2306,10 +2309,10 @@ Options for the linkcheck builder
 
 .. confval:: linkcheck_anchors_ignore
 
-   A list of regular expressions that match URIs that should skip checking
-   the validity of anchors in links. This allows skipping entire sites, where
-   anchors are used to control dynamic pages, or just specific anchors within
-   a page, where javascript is used to add anchors dynamically, or use the
+   A list of regular expressions that match URIs that should skip checking the
+   validity of anchors in links. This allows skipping entire sites, where
+   anchors are used to control dynamic pages, or just specific anchors within a
+   page, where javascript is used to add anchors dynamically, or use the
```

Comment:
here again the original paragraph had shorter lines, and overall had more equilibrium